keanacobarde/on-paper
# SIGN-IN REVAMP

## Description
- Altered template for sign-in page and re-implemented sign-in functionality.

## Related Issue
#45 

## Motivation and Context
This is an aesthetic change that heightens the UX of the site. Having a semi-realistic looking sign-in page aids in the first impression a user has of the site - not necessary for functionality but a nice touch. 

## How Can This Be Tested?
- Pull PR branch
- Install required dependencies
- Stand server. The very first page that appears should be the sign-in page. 
- Select the 'Sign-in' button to ensure that the functionality works.

## Screenshots (if appropriate):
N / A

## Types of changes
